internal/pkg/cli/svc_init.go

```diff
@@ -371,6 +376,25 @@ func (o *initSvcOpts) askSvcPort() error {
 	return nil
 }
 
+func (o *initSvcOpts) parseHealthCheck() (*manifest.ContainerHealthCheck, error) {
+	o.setupParser(o)
+	hc, err := o.df.GetHealthCheck()
+	if err != nil {
+		return nil, fmt.Errorf("get healthcheck from Dockerfile: %w", err)
```

Comment:
Also maybe do we want to add dockerfile name at the end of it? Maybe `...from Dockerfile %s, %w`